src/lib/utils/prefixKeys.ts

```diff
@@ -2,7 +2,7 @@ import mapKeys from 'lodash/mapKeys';
 
 function prefixKeys(prefix: string, obj: object) {
   if (obj) {
-    return mapKeys(obj, (_value, key) => prefix + key);
+    return mapKeys(obj, (_0, key) => prefix + key);
```

Comment:
_value was fine no?

Reply:
TSLint can't ignore underscore leading parameters right now. It complains when not following the format `_{number}`.

Reference: https://github.com/palantir/tslint/issues/3442.